loc/en.js

```diff
@@ -23,6 +23,33 @@ module.exports = {
     scan: 'Scan',
     storage_is_encrypted: 'Your storage is encrypted. Password is required to decrypt it',
   },
+  airdrop: {
+    title: 'Airdrop',
+    itIsAlive: {
+      subtitle: 'Airdrop is live! Check it out!',
+      description: 'We are happy to announce our first Bitcoin Vault Airdrop for Gold Wallet.',
+      iDontCare: "I don't care",
+      learnMore: 'Learn more',
+    },
+    requirements: {
+      soundsGreat: 'Sounds great!',
+      subtitle: 'Thank you for creating the Bitcoin Vault community.',
+      description:
```

Comment:
I would add the amount 250.000 as a variable to change in an easy way for all translations if needed

Reply:
done